### DISCUSSION TITLE
Turning off notifications

### DISCUSSION THREAD

**Discussion by polishdataengineer** [writer/writer-framework#156] [Q&A]
Is there a way to force the notifications to stop?
<img width="204" alt="image" src="https://github.com/streamsync-cloud/streamsync/assets/69598075/6701c192-b851-4ded-bb88-30ac9db412b0">

**Comment by FabienArcellier** [comment]
A simpleway to do that is to add a css rule. I don't think there is a way on backend or on ui designer to suspend notification.

```css
.notifications {
  display: none;
}
```

A good option would be to add a property on Root to select how to display notification (UI, dev console, none, ...). It would be perfect for me if we can bind this property on a backend side.
